### PR TITLE
LJE-80/User Profile Page

### PR DESCRIPTION
**Pull request by zhuTRN**

### CODE REVIEW

**Comment by zhuTRN** [comment]
Hey Fahad, starting from Low fidelity Implementation they are my commits to the user profile page, please let me know if there is any problems.

File: packages/frontend/src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -36,26 +36,26 @@ const Category = styled.p`
...
-                  <Category>{item.title}</Category>
-                </Link>
-              </li>
-            );
-          })}
-        </NavMenuInactive>
-      </nav>
-    </NavbarItems>
-  );
+    return (
```
Comment (fhdamd) [inline packages/frontend/src/components/Navbar/Navbar.tsx:39]: Hi Brian, can you please check your tab size and confirm it is set to 2 in your VS Code settings? Seems like if you make a minor change in your file it seems to auto-format the file according to your tab settings.

File: packages/frontend/src/components/ProductCard.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        display:inline-block;
+        vertical-align: top;
+    }
+    }
+`
+
+
+
+const ProductCard = () => {
+    const listItems = ProductData.map((item: any) =>
```
Comment (fhdamd) [inline packages/frontend/src/components/ProductCard.tsx:65]: We shouldn't have the type of any object or variable as `any`. Can we define the right types for the `item` object?

File: packages/frontend/yarn.lock [hunk 3]
```diff
@@ -939,10 +939,17 @@
     "@babel/helper-validator-option" "^7.16.7"
     "@babel/plugin-transform-typescript" "^7.16.7"
 
+<<<<<<< HEAD
+"@babel/runtime@^7.12.5", "@babel/runtime@^7.13.10", "@babel/runtime@^7.16.7", "@babel/runtime@^7.4.4", "@babel/runtime@^7.5.5", "@babel/runtime@^7.7.2", "@babel/runtime@^7.7.6", "@babel/runtime@^7.8.4", "@babel/runtime@^7.8.7":
```
Comment (fhdamd) [inline packages/frontend/yarn.lock:943]: this is again a corrupted `yarn.lock` file. Please regenerate the file properly.

**Review by fhdamd — CHANGES_REQUESTED** [review]

**Comment by fhdamd** [comment]
Hi Brian,please review the changes requested and resolve the merge conflicts appropriately.

Thanks and good work.

**Review by fhdamd — APPROVED** [review]

**Comment by fhdamd** [comment]
Hi Brian can you please resolve the merge conflicts? I have approved your changes.

**Comment by zhuTRN** [comment]
Hey @fhdamd , I am able to see the whole frontend when I yarn start, but for some reason, the deploy preview just doesn't display anything. It still says SignIn is not defined if you go to the deploy preview.
 Do I just merge it since it is working on the development server?

**Pull request merged**